web/src/compose_tooltips.ts

```diff
@@ -289,3 +292,70 @@ export function hide_compose_control_button_tooltips($row: JQuery): void {
         this._tippy?.hide();
     });
 }
+
+export function show_go_to_conversation_button_intro_tooltip(): void {
+    const $go_to_conversation_button = $(".conversation-arrow");
+    const element: tippy.ReferenceElement = the($go_to_conversation_button);
+
+    const tippy_instance =
+        element._tippy ??
+        tippy.default(element, {
+            hideOnClick: false,
+            trigger: "manual",
+            appendTo: document.body,
+        });
+    tippy_instance.setContent(
+        parse_html($("#compose_go_to_conversation_button_tootltip_template").html()),
+    );
+
+    tippy_instance.show();
+}
+
+export let can_show_go_to_conversation_button_intro_tooltip = (): boolean => {
+    // The tooltip is shown only if:
+    // The tooltip hasn't been shown before.
+    // The user has confirmed a topic recipient change to a topic that is not the current view.
+    // The "Go to conversation" button is active.
+    // There are no compose banners currently visible.
+    // The user hasn't clicked the button (which dismisses the tooltip).
+
+    // This helps in updating tooltip when switching between topics.
+    hide_go_to_conversation_button_intro_tooltip();
+
+    if (
+        !onboarding_steps.ONE_TIME_NOTICES_TO_DISPLAY.has("intro_go_to_conversation_button_tooltip")
+    ) {
+        return false;
+    }
```

Comment:
I think we should call `hide_go_to_conversation_button_intro_tooltip` after the `onboarding_step` check here as well, unless there's a specific case you are trying to avoid.

Reply:
I must have missed it, I apologize for the inconvenience caused. It is fixed in the subsequent push.